principles/index.html

```diff
@@ -15,6 +15,13 @@
           companyURL: 'https://nytimes.com/',
           url: 'https://berjon.com/',
           w3cid: 34327,
+        },
+                {
+          name: 'Lukasz Olejnik',
+          company: 'Independent researcher (Invited Expert)',
```

Comment:
I'm going to suggest that we drop "Invited Expert" here because the CGs permit individual participation, i.e., there's really not an "Independent researcher" here.

```suggestion
          company: 'Independent researcher',
```

Reply:
Independent researcher, then, without IE. Thanks.